static/js/cf-frontend.js

```diff
@@ -360,7 +360,9 @@ function codePermanentLinkGeneration() {
   var inputURL = encodeURI(input);
   inputURL = "&code=" + inputURL;
 
-  var curUrl = window.location.host + '/';
+  // TODO: "/live" is needed for eisop. What is the proper way
+  // to look up host and directory?
+  var curUrl = window.location.host + "/live/";
```

Comment:
The `location.host` only generated `eisop.uwaterloo.ca` which then meant that the links didn't work.
Is there a way to look up the complete URL from which the script is running?

Reply:
I think `window.location.href` would get the current page url instead of just the host name.

I've fixed this, and also the indentation problem by #33 .